src/action-creators/middlewareCreateAction.js

```diff
@@ -25,6 +26,10 @@ export default (
     return nestAction(pathname, action, prevLocation, history, kind)
   }
   catch (e) {
+    if (__DEV__) {
+      console.error(e)
```

Comment:
I think some context would be helpful, especially for non-maintainer user who accidentally see it.

Reply:
what do you think about `console.error("[redux-first-router] Internal exception when parsing action, fallback to NOT_FOUND. Original exception: ", e");`